spec/es5.d.ts

```diff
@@ -0,0 +1,13 @@
+interface Symbol { }
```

Comment:
Basically because specs will fail to compile without it, and there is no way to indicate in `tsconfig.json` to add one additional file, without declaring all the files.  The easiest place was to add it in the specs folder, and then reference it from the builds.